packages/web/src/server/index.ts

```diff
@@ -98,12 +107,26 @@ export async function track(
 
     const url = new URL(origin);
 
+    let flagsToReport: Record<string, unknown> = {};
+    const allFlags = requestContext?.flags?.getValues();
```

Comment:
nit for readability

I'd rename:
- `flagsToReport` -> `flagValuesToReport`
- `allFlags` -> `allFlagValues`

This avoids using the ambiguous term "flag" which can refer to the key of a flag, or the key/value of a flag.